src/common/ui/setting-input.vue

```diff
@@ -0,0 +1,54 @@
+<template>
```

Comment:
`setting-input` is unused now so it's probably better to save this file locally until it's needed in the future?

Reply:
Sure